test_cromwell.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+shutdown() {
+    cd $INITIAL_DIR
+    pkill -P $$
+    exit 0
+}
+
+trap "shutdown" EXIT
+
+PROGNAME=`basename $0`
+
+usage="
+$PROGNAME [-b branch] [-r rundir] [-c config file] [-p parallelism factor]
+
+Builds and runs specified branch of Cromwell and runs Centaur against it.
+
+Arguments:
+    -b    Branch of Cromwell to test
+    -r    Directory where script is run (defaults to current directory)
+    -c    If supplied, the config file to pass to Cromwell
+    -p    Number of simultaneous tests to run. Defaults to 3
+"
+
+INITIAL_DIR=`pwd`
+
+PARALLELISM_FACTOR=3
+CROMWELL_BRANCH="develop"
+RUN_DIR=`pwd`
+CONFIG_STRING=""
+
+while getopts ":hb:r:c:p:" option; do
+    case "$option" in
+	h) echo "$usage"
+	   exit
+	   ;;
+	b) CROMWELL_BRANCH=$OPTARG
+	   ;;
+        r) RUN_DIR=$OPTARG
+	   mkdir -p $RUN_DIR
+	   ;;
+        c) CONFIG_STRING="-Dconfig.file=$OPTARG"
+	   ;;
+        p) PARALLELISM_FACTOR=$OPTARG
+	   ;;
+	:) printf "Missing argument for -%s\n" "$OPTARG" >&2
+	   echo "$usage" >&2
+	   exit 1
+	   ;;
+       \?) printf "Illegal option: -%s\n" "$OPTARG" >&2
+           echo "$usage" >&2
+           exit 1
+	   ;;
+    esac
```

Comment:
I nearly used this as an excuse to finally put hands to keyboard with my goal of learning Elixir, so you should be thankful the 1980s are calling here